scheduler/playbooks/k8s_add_node.yml

```diff
@@ -54,23 +59,25 @@
           - host: https://mirror.gcr.io
             capabilities: ["pull", "resolve"]
             skip_verify: false
-    kubeadm_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/kubeadm.tar.gz"
-    kubectl_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/kubectl.tar.gz"
-    kubelet_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/kubelet.tar.gz"
-    crictl_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/cri-tools-v1.26.1.tar.gz"
-    calicoctl_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/calicoctl-v3.25.2.tar.gz"
-    calico_crds_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/calicocrds-v3.25.2.tar.gz"
+    kubeadm_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/kubeadm-{{ hostvars['localhost']['k8s_version'] }}.tar.gz"
+    kubectl_download_url: "{{ hostvars['localhost']['offline_tarball_path'] }}/kubectl-{{ hostvars['localhost']['k8s_version'] }}.tar.gz"
```

Comment:
we have not changed package name in k8s.json, hence here also need to keep same name

![image](https://github.com/user-attachments/assets/ee270d85-70bd-49aa-88e3-60ec859c8633)